src/views/Farms/components/FarmCard/DetailsSection.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import useI18n from 'hooks/useI18n'
+import styled from 'styled-components'
+import { Text, Flex, Link, OpenNewIcon } from '@pancakeswap-libs/uikit'
+import getLiquidityUrlPathParts from 'utils/getLiquidityUrlPathParts'
+
+export interface TokenAddressesObject {
+  56?: string
+  97?: string
+}
+export interface ExpandableSectionProps {
+  bscScanAddress?: string
+  removed?: boolean
+  totalValueFormated?: string
+  lpLabel?: string
+  quoteTokenAdresses?: TokenAddressesObject
+  quoteTokenSymbol?: string
+  tokenAddresses: TokenAddressesObject
+}
+
+const Wrapper = styled.div`
+  margin-top: 24px;
+`
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  font-weight: normal;
+  color: ${({ theme }) => theme.colors.text};
+  display: flex;
+  align-items: center;
+
+  svg {
+    padding-left: 4px;
+    height: 18px;
+    width: auto;
+    fill: ${({ theme }) => theme.colors.primary};
+  }
+`
+
+const DetailsSection: React.FC<ExpandableSectionProps> = ({
+  bscScanAddress,
+  removed,
+  totalValueFormated,
+  lpLabel,
+  quoteTokenAdresses,
+  quoteTokenSymbol,
+  tokenAddresses,
+}) => {
+  const TranslateString = useI18n()
+  const liquidityUrlPathParts = getLiquidityUrlPathParts({ quoteTokenAdresses, quoteTokenSymbol, tokenAddresses })
+
+  return (
+    <Wrapper>
+      <Flex justifyContent="space-between">
+        <Text>{TranslateString(316, 'Stake')}:</Text>
+        <StyledLink external href={`https://exchange.pancakeswap.finance/#/add/${liquidityUrlPathParts}`}>
```

Comment:
You have a component doing this here https://github.com/pancakeswap/pancake-uikit/blob/master/src/components/Link/LinkExternal.tsx#L6